lib/src/components/Modal/modal.module.scss

```diff
@@ -154,4 +109,102 @@
     margin-bottom: (var(--spacing-0));
     margin-top: var(--spacing-md);
   }
+
+  .root {
```

Comment:
Camille commented on a PR of mine that it'd be good to have `.root` at the top for consistency and I agree. 🤓